csrc/codegen.cpp

```diff
@@ -2972,6 +2960,10 @@ class CudaKernelGenerator : private kir::ConstIrVisitor {
           aligned_array_of_regs_.insert(tv);
         }
       }
+      // tv is aligned if alias is aligned
```

Comment:
Why is this? I understand this is just a naming difference, but whether the original allocation is aligned or not shouldn't matter for the aliasing tensor, right? For example, this `tv` can be a tensor with no alignment requirement, right?

Reply:
On the other hand, `T29` does't have to be aligned. We can use dynamic cast to remove this. But then we need to change the code of alias allocation.